src/main/java/net/tiagofar78/prisonescape/managers/MessageLanguageManager.java

```diff
@@ -104,6 +107,14 @@ public String getSuccessfullyLeftGameMessage() {
 	public String getSuccessfullyForceStoppedGameMessage() {
 		return _successfullyForceStoppedGameMessage;
 	}
+
+	public String getSuccessfullyRejoinedGameMessage() {
+		return _successfullyRejoinedGameMessage;
+	}
+
+	//	########################################
```

Comment:
Announcements are just warnings broadcasted to everyone so I don't think this division is really necessary